zipkin-storage/cassandra/src/main/java/zipkin/storage/cassandra/CassandraStorage.java

```diff
@@ -220,11 +220,11 @@ public Session session() {
     for (String cf : ImmutableList.of(
         "traces",
         "dependencies",
-        "service_names",
-        "span_names",
-        "service_name_index",
-        "service_span_name_index",
-        "annotations_index",
+        Tables.SERVICE_NAMES,
+        Tables.SPAN_NAMES,
+        Tables.SERVICE_NAME_INDEX,
+        Tables.SERVICE_SPAN_NAME_INDEX,
+        Tables.ANNOTATIONS_INDEX,
         "span_duration_index"
```

Comment:
not documenting this one, yet, as it has a whole lot of analysis pending per https://github.com/openzipkin/zipkin-java/issues/200